apps/dashboard/src/components/workflow-editor/steps/digest/digest-window.tsx

```diff
@@ -56,62 +40,92 @@ export const DigestWindow = () => {
           <span>Digest window</span>
         </span>
       </FormLabel>
-      <Tabs defaultValue="editor" className="flex h-full flex-1 flex-col" value="editor">
+      <Tabs
+        value={digestType}
+        className="flex h-full flex-1 flex-col"
+        onBlur={(e) => {
+          e.preventDefault();
+          e.stopPropagation();
+        }}
+        onValueChange={async (value) => {
+          setDigestType(value);
+          if (value === SCHEDULED_DIGEST_TYPE) {
+            setValue(AMOUNT_KEY, undefined, { shouldDirty: true });
+            setValue(UNIT_KEY, undefined, { shouldDirty: true });
+            setValue(CRON_KEY, EVERY_MINUTE_CRON, { shouldDirty: true });
+          } else {
+            setValue(AMOUNT_KEY, '', { shouldDirty: true });
+            setValue(UNIT_KEY, TimeUnitEnum.SECONDS, { shouldDirty: true });
+            setValue(CRON_KEY, undefined, { shouldDirty: true });
+          }
+          await trigger();
+          saveForm();
```

Comment:
When switching the digest tabs, fill the form with the default values, trigger the validation, and save the form.

Reply:
I suggest adding a confirmation popup window when switching the digest tabs.
especially when switching from "schedule" to "regular"
As someone might mis-click it and lose all the progress saved in the scheduled form.


https://github.com/user-attachments/assets/13081bea-4864-49f2-9d94-ce73733ae8c1